pkg/proxy/nftables/proxier.go

```diff
@@ -686,6 +698,14 @@ func (proxier *Proxier) setupNFTables(tx *knftables.Transaction) {
 			),
 		})
 	}
+
+	// Create empty containers + flush
+	proxier.clusterIPs = newNFTElementStorage("set", clusterIPsSet, tx)
+	proxier.serviceIPs = newNFTElementStorage("map", serviceIPsMap, tx)
+	proxier.firewallIPs = newNFTElementStorage("map", firewallIPsMap, tx)
+	proxier.noEndpointServices = newNFTElementStorage("map", noEndpointServicesMap, tx)
+	proxier.noEndpointNodePorts = newNFTElementStorage("map", noEndpointNodePortsMap, tx)
+	proxier.serviceNodePorts = newNFTElementStorage("map", serviceNodePortsMap, tx)
```

Comment:
https://github.com/kubernetes/kubernetes/pull/126013/commits/cb279fbf4f6697c6edf07b0432e8609159417868#r1682929066
so, you can initialize these objects only at the beginning and here just do something like
```
proxier.clusterIPs.reset() // calls Clear() on the object
proxier.clusterIPs.update(clusterIPsSet, tx) 
```


```

and

```

Reply:
that is a good idea!